source/codegen/templates/service_helpers.mako

```diff
@@ -187,6 +187,8 @@ one_of_case_prefix = f'{namespace_prefix}{function_name}Request::{PascalFieldNam
 % elif c_type == 'ViSession':
       auto ${parameter_name}_grpc_session = ${request_snippet};
       ${c_type} ${parameter_name} = session_repository_->access_session(${parameter_name}_grpc_session.id(), ${parameter_name}_grpc_session.name());\
+% elif c_type == 'ViInt32[]':
```

Comment:
Note: Other option is adding if condition inside the elif statement below which would look like this:

![alternative](https://user-images.githubusercontent.com/77618357/114875530-00396880-9e1b-11eb-9a00-709b9051b078.PNG)

I used the current approach to avoid extra branching. But if owners feel that this alternative approach is easy to read, then I will make that change

Reply:
What you have looks fine, except I would like there to be only one `reinterpret_cast` without the `const`. The double-cast doesn't add any value.

When you merge with PR #7, you'll put both `ViInt32[]` and `ViAttr[]` on the same line, right?

Reply:
@jonlugoatx ,
>  I would like there to be only one reinterpret_cast without the const.

We initially tried this way. But we got the following error saying that "reinterpret_cast cannot cast away const or other type qualifiers".

![Comment_const](https://user-images.githubusercontent.com/77618357/114977693-b8135800-9ea5-11eb-92f7-c5b5919ede16.png)

Then @ganeshnie suggested using const_cast before reinterpret_cast.

> When you merge with PR #7, you'll put both ViInt32[] and ViAttr[] on the same line, right?

Yes

Reply:
@jonlugoatx 
I discussed with @ganeshnie and we decided to resolve this comment and submit PR for now. Later if we need any changes, we will do it as part of separate PR.